packages/medusa-plugin-sendgrid/src/services/sendgrid.js

```diff
@@ -579,20 +581,19 @@ class SendGridService extends NotificationService {
     const giftCard = await this.giftCardService_.retrieve(id, {
       relations: ["region", "order"],
     })
-
-    if (!giftCard.order) {
-      return
-    }
-
+    // Should this use the custom Tax Provider if any?
     const taxRate = giftCard.region.tax_rate / 100
-
-    const locale = await this.extractLocale(order)
+    // get locale from the order - but maybe we should get this from the region?
+    const locale = giftCard.order ? await this.extractLocale(order) : null;
+    // if the gift card has no order, we still return the email associated to the gift card (created from admin panel).
+    const email = giftCard.order ? giftCard.order.email : giftCard.metadata.email;
 
     return {
       ...giftCard,
       locale,
-      email: giftCard.order.email,
-      display_value: giftCard.value * (1 + taxRate),
+      email,
+      display_value: (giftCard.value * (1 + taxRate) / 100).toFixed(2),
```

Comment:
todo: Use util to cover all currency cases
```suggestion
      display_value: `${this.humanPrice_((giftCard.value * 1+ taxRate), giftCard.region.currency_code)} ${giftCard.region.currency_code}`,
```